bootcamp/tutorials/quickstart/multimodal_rag_with_milvus.ipynb

```diff
@@ -0,0 +1,884 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    !pip install --upgrade pymilvus Pillow numpy opencv-python datasets openai
</pre><p>remove numpy since it has been already required by pymilvus</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1398/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>